src/libraries/Microsoft.Extensions.FileSystemGlobbing/ref/Microsoft.Extensions.FileSystemGlobbing.cs

```diff
@@ -81,14 +81,14 @@ public partial class FileInfoWrapper : Microsoft.Extensions.FileSystemGlobbing.A
         public FileInfoWrapper(System.IO.FileInfo fileInfo) { }
```

Comment:
Should the implementation of this ctor check for `null` and throw an ArgNullException?